src/algorand_transactions/transaction.rs

```diff
@@ -783,7 +803,7 @@ mod tests {
             .unwrap()[0]
             .clone();
         let result = tx.to_id().unwrap();
-        let expected_result = "NIS7GP6IZPCJMTZBDCVOC4RCOA4J33ZLDGH26Y7N2ZYRRAAQLF5A";
+        let expected_result = "5IBWPOEE3ZB7UBO3G42T34YVPVK6CT32T46HHECESD5BJC5SVK3A";
```

Comment:
Note that the correct hash to test was not NIS7GP6IZPCJMTZBDCVOC4RCOA4J33ZLDGH26Y7N2ZYRRAAQLF5A, but 5IBWPOEE3ZB7UBO3G42T34YVPVK6CT32T46HHECESD5BJC5SVK3A which is the tx containing the inner txs

Reply:
Noted. Now we know that the inner txs use the tx ID of their parent!